docs/integrations.md

```diff
@@ -31,3 +31,13 @@ __Link:__ [https://github.com/adfinis-sygroup/aem-docker/tree/master](https://gi
 __Description:__  "Download and unarchive the latest kompose release asset for your OS"
 
 __Link:__ [https://github.com/chouseknecht/kompose-install-role](https://github.com/chouseknecht/kompose-install-role)
+
+### Fabric8 Maven Plugin by Red Hat
+
+__Description:__  "The fabric8-maven-plugin is used to bring your Java applications onto Kubernetes or Openshift. The main task of this plugin is to build Docker Images and generate Kubernetes or  Openshift resource descriptors. A docker-compose configuration is one of the method to bring up docker compose deployments on a Kubernetes/Openshift cluster. Fabric8-maven Plugin processes the external docker-compose file and generates Kubernetes/Openshift resources using Kompose."
```

Comment:
Openshift -> OpenShift
Images -> images
There's an extra spec between `or  Openshift`
`A Docker Compose configuration is one of the methods to bring up deployments on a Kubernetes or OpenShift cluster`
Fabric8-maven -> Fabric8 Maven Plugin
`The Fabric8-maven plugin processes the external `docker-compose.yaml` file and generates Kubernetes or OpenSHift resources via Kompose.`

Please separate sentences too, it's quite long. Markdown doesn't cut-off sentences if there is a newline.